docs/resources/server.md

```diff
@@ -164,6 +164,7 @@ resource "random_password" "server_image_password" {
 - `image_password` - (Optional)[string] Required if `ssh_key_path` is not provided.
 - `type` - (Optional)[string] Server usages: [ENTERPRISE](https://docs.ionos.com/cloud/compute-engine/virtual-servers/virtual-servers) or [CUBE](https://docs.ionos.com/cloud/compute-engine/virtual-servers/cloud-cubes). This property is immutable.
 - `label` - (Optional) A label can be seen as an object with only two required fields: `key` and `value`, both of the `string` type. Please check the example presented above to see how a `label` can be used in the plan. A server can have multiple labels.
+- `delete_volumes` - (Optional) [bool] deleting all the volumes of a server even if no boot_volume provided
```

Comment:
@gabi-ionos it shouldn't be a space between `(Optional)` and `[bool]`, it should be `(Optional)[bool]`. 

Also, I would change this description since it makes me think that if `boot_volume` is provided, all volumes will be deleted which it's false. 

I suggest something like: `(Optional)[bool] Enforce the deletion of all volumes of a server, not only boot_volume.` This also makes clear that if `delete_volumes` is `false`, only the `boot_volume` will be deleted (this happens now with our current code but I will describe what I think related to this behavior in another comment).

Reply:
@cristiGuranIonos we removed `delete_volumes` attribute, so this comment is no longer available.